.freeipa-pr-ci.yaml

```diff
@@ -13,172 +13,172 @@ topologies:
     memory: 6700
 
 jobs:
-  fedora-27/build:
+  fedora-rawhide/build:
     requires: []
     priority: 100
     job:
       class: Build
       args:
         git_repo: '{git_repo}'
         git_refspec: '{git_refspec}'
-        template: &ci-master-f27
-          name: freeipa/ci-master-f27
+        template: &ci-master-frawhide
+          name: freeipa/ci-master-frawhide
           version: 1.0.3
```

Comment:
Hello!

For a `frawhide` image existing available versions are 0.0.2, 0.0.3, 0.0.4

Would you mind to change that?

Reply:
sure. thank you.